src/main/scala/mesosphere/marathon/state/Migration.scala

```diff
@@ -151,6 +154,38 @@ class Migration @Inject() (
   }
 }
 
+object LegacyTaskMigrationHelper {
+
+  private[this] val log = Logger.getLogger(getClass.getName)
+
+  // TODO is this still needed? it was introduced in 4c1c5d994aa303a90d54985efb8490e6cc69dd65 on 08/07/2014 ...
```

Comment:
Yes, that one is replaced by a taskRepo dependency. I added functionality to prevent migration from old versions – that is currently set to 0.3.0; I guess we have no knowledge of whether a migration would work from < 0.3.0.